src/Disqord.Core/Enums/Permission.cs

```diff
@@ -181,18 +184,23 @@ public enum Permission : ulong
         ManageThreads = 1ul << 34,
 
         /// <summary>
-        ///     Allows using public threads.
+        ///     Allows creating public threads.
         /// </summary>
-        UsePublicThreads = 1ul << 35,
+        CreatePublicThreads = 1ul << 35,
 
         /// <summary>
-        ///     Allows using private threads.
+        ///     Allows creating private threads.
         /// </summary>
-        UsePrivateThreads = 1ul << 36,
+        CreatePrivateThreads = 1ul << 36,
 
         /// <summary>
         ///     Allows using stickers from other guilds.
         /// </summary>
-        UseExternalStickers = 1ul << 37
+        UseExternalStickers = 1ul << 37,
+
+        /// <summary>
+        ///     Allows sending messages in threads.
+        /// </summary>
+        SendMessagesInThreads = 1ul << 38,

```

Comment:
```suggestion
        SendThreadMessages = 1ul << 38,
```
Would this name fit better?